aytests/self_update_reg.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e -x
+
+# FATE#319716 - the installer updates need to be installed into inst-sys
+# check that the URL from the registration server is detected.
+zgrep "Updates for 'SLES' are available at.*static/repos.*" /var/log/YaST2/y2log*
```

Comment:
Hm, be careful with zgrep. See:
https://github.com/yast/aytests-tests/pull/27/files